ci/infra/testrunner/utils/constants.py

```diff
@@ -83,6 +83,7 @@ def __init__(self):
             super().__init__()
             self.binpath = None
             self.srcpath = None
+            self.verbosity = 2
```

Comment:
shouldnt this read from the skuba conf?

Reply:
I think this is here in case there is an outdated `var.yaml` file that doesn't have a value.

Reply:
Yeah; this is a default that gets overridden by the config value if present.